action/login/login.go

```diff
@@ -94,6 +94,10 @@ func (c *Config) Tokens(client *vela.Client, addr string) error {
 		// log the response object for troubleshooting purposes
 		logrus.Debug(resp)
 
+		if err != nil {
```

Comment:
you are already in a `err != nil` block at this point. did you mean `resp != nil` ?

Reply:
Attempting to make fixes on Monday morning is challenging... I've updated it to check for the nil response.